test/cli/test/manifests/verify-signature/policies.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: validate-yaml
+spec:
+  validationFailureAction: enforce
+  background: false
+  webhookTimeoutSeconds: 30
+  failurePolicy: Fail  
+  rules:
+    - name: validate-yaml
+      match:
+        resources:
```

Comment:
The new schema version going forward will require `any` or `all` under `match`/`exclude` blocks, so suggest you go ahead and add that now.

Reply:
@chipzoller Thank you for your comment. I have updated policies.